package.json

```diff
@@ -29,11 +29,35 @@
 		"redhat.vscode-quarkus",
 		"redhat.vscode-kaoto"
 	],
+	"extensionPackTitles": [
```

Comment:
it was much easier in first iteration to add custom section with extension titles, because ExTester is currently limited to search for extensions just using titles 🙁

Reply:
This config is not part of the documentation:
https://code.visualstudio.com/api/references/extension-manifest

Can you precise what is the impact?
Will wee need to maintain a map with the extensionPack list?

Reply:
there is possibility to provide custom sections into package.json.. so no impact for extension itself.. but yes, we will need to maintain also the names of extensions. .. till someone (probably me..) will implement the logic for ExTester to be also able to find extensions using extensionID and not only by using names as it is now